context/app/static/js/components/Detail/FileBrowser/FileBrowser.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react';
+import FileBrowserNode from '../FileBrowserNode';
+import { ScrollPaper } from './style';
+
+function FileBrowser(props) {
+  const { files } = props;
+  const [fileTree, setFileTree] = useState({});
+  useEffect(() => {
+    const treePath = {};
+    files.forEach((f) => {
+      const levels = f.rel_path.split('/');
+      const file = levels.pop();
+
+      const fileObj = { file, fullPath: f.rel_path, description: f.description, edam_term: f.edam_term, size: f.size };
+
+      if (levels.length === 0) {
+        if ('files' in treePath) {
+          treePath.files.push(fileObj);
+        } else {
+          treePath.files = [fileObj];
```

Comment:
Instantiating `treePath.files = []` above would make this a little clearer since you could just `.push`, I think.